tools/system_libs.py

```diff
@@ -616,9 +607,21 @@ class Dummy(object):
 
   libs_to_link = []
   already_included = set()
-
   system_libs_map = {l.shortname: l for l in system_libs}
 
+  # Setting this in the environment will avoid checking dependencies and make building big projects a little faster
+  # 1 means include everything; otherwise it can be the name of a lib (libc++, etc.)
+  # You can provide 1 to include everything, or a comma-separated list with the ones you want
+  force = os.environ.get('EMCC_FORCE_STDLIBS')
+  if force == '1':
+    force_all = True
```

Comment:
I don't think you need the `force_all` variable anymore since adding the lib to `force` has the same effect.